test/check-verify

```diff
@@ -60,8 +60,7 @@ def run(opts, sink):
 
 def main():
     parser = testlib.arg_parser()
-    parser.add_argument('--publish', dest='publish', action='store',
-                        help='Publish results centrally to a sink')
+    parser.add_argument('--publish', action='store', help="Unused")
```

Comment:
That's in github-task. ... check-verify no longer has anything to do with publishing results.